src/test/java/com/rits/tests/cloning/TestCloner.java

```diff
@@ -796,6 +796,7 @@ public void testEnumMapWithNullValue() {
 		originalMap.put(TestEnum.A, null);
 
 		EnumMap<TestEnum, String> clonedMap = cloner.deepClone(originalMap);
+		assertTrue("Cloned LinkedHashSet not instanceof EnumMap", clonedMap instanceof EnumMap);

```

Comment:
Why do we need this assertion here? If clonedMap wasn't an EnumMap, wouldn't we get a class cast exception on previous line (798)?

Reply:
Well, reading that description I just copy-pasted it from the lines below. :) I agree this is redundant, I'll remove that.

Reply:
Fixed.